tasmota/tasmota_template.h

```diff
@@ -101,6 +101,7 @@ enum UserSelectablePins {
   GPIO_ADC_BUTTON, GPIO_ADC_BUTTON_INV,  // Analog Button
   GPIO_ADC_RANGE,                      // Analog Range
   GPIO_ADC_CT_POWER,                   // ANalog Current
+  GPIO_ADC_PH,                         // Analog PH Sensor
```

Comment:
Inserting it here will break every current installation. Needs to go at the end of the list.